eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/legacyjpa/EmbeddedEventStore.java

```diff
@@ -521,6 +522,12 @@ public Builder spanFactory(@Nonnull SpanFactory spanFactory) {
             return this;
         }
 
+        @Override
```

Comment:
You didn't change the JavaDoc of the `Builder` class and `builder()` method in accordance with this change.